### ISSUE TITLE
font fallback is stripped from .import file when importing fontdata

### ISSUE THREAD

**Issue by jamie-pate** [godotengine/godot#92297]
### Tested versions

- Reproducible in Godot 4.3-dev6
- Reproducible in Godot 4.2.2

### System information

Godot v4.3.dev6 - Ubuntu 22.04.4 LTS 22.04 - X11 - Vulkan (Forward+) - dedicated NVIDIA GeForce RTX 3050 Laptop GPU () - 12th Gen Intel(R) Core(TM) i7-12700H (14 Threads)

### Issue description

The Advanced Import Settings for otf and ttf files allows you to specify fallback fonts, but whenever you load the project they will be deleted.

```
$ ~/bin/Godot_v4.3-dev6_linux.x86_64 -e --path fontfile_fallback/
Godot Engine v4.3.dev6.official.64520fe67 - https://godotengine.org
Vulkan 1.3.242 - Forward+ - Using Device #2: NVIDIA - NVIDIA GeForce RTX 3050 Laptop GPU

ERROR: Cannot open file 'res://.godot/imported/NotoSansCJKjp-Regular.otf-a610d2c3227669d49989938924363884.fontdata'.
   at: load (core/io/resource_format_binary.cpp:1189)
ERROR: Failed loading resource: res://.godot/imported/NotoSansCJKjp-Regular.otf-a610d2c3227669d49989938924363884.fontdata. Make sure resources have been imported by opening the project in the editor at least once.
   at: _load (core/io/resource_loader.cpp:281)
ERROR: Failed loading resource: res://NotoSansCJKjp-Regular.otf. Make sure resources have been imported by opening the project in the editor at least once.
   at: _load (core/io/resource_loader.cpp:281)
ERROR: ConfigFile parse error at res://Lato-Regular.ttf.import:27: Can't load resource at path: 'res://NotoSansCJKjp-Regular.otf'..
   at: _parse (core/io/config_file.cpp:304)
WARNING: res://font_variation.tres:3 - ext_resource, invalid UID: uid://v80au4dacx41 - using text path instead: res://Lato-Regular.ttf
     at: load (scene/resources/resource_format_text.cpp:457)
WARNING: res://font_variation.tres:3 - ext_resource, invalid UID: uid://v80au4dacx41 - using text path instead: res://Lato-Regular.ttf
     at: load (scene/resources/resource_format_text.cpp:457)

```
Seems like it's due to a dependency created by adding fonts to the `fallbacks` array... if the other font resources haven't been imported yet they are removed from the fallbacks array? 

Some properties in godot4 are being saved this way, rather than `fallbacks/0 = ExtResource(...)` which I think is causing a bunch of issues. (the `scene.gltf.import`  `_subresources` property also fails to be updated across renames!)

### Steps to reproduce

Load the reproduction project. `Lato-Regular.ttf.import` will be modified and it's fallbacks set to [] 
it should remain as
`fallbacks=[Resource("res://NotoSansCJKjp-Regular.otf"), Resource("res://DroidSansFallback.ttf")]`

### Minimal reproduction project (MRP)

[fontfile_fallback.zip](https://github.com/godotengine/godot/files/15425692/fontfile_fallback.zip)

**Comment by aaronfranke** [comment]
Likely related, even when the fallback is valid, Godot complains that it has an invalid UID (while showing the correct UID).

```
WARNING: res://visual/fonts/FontMono.ttf: In external resource #0, invalid UID: uid://clpf84p1lfwlp - using text path instead: res://visual/fonts/Font.ttf
```

The main font has this: `fallbacks=[Resource("res://visual/fonts/Font.ttf")]`

The `res://visual/fonts/Font.ttf` file has this: `uid="uid://clpf84p1lfwlp"`

**Comment by Hilderin** [comment]
Closed by #92667

**Comment by TinyTakinTeller** [comment]
Only workaround for now is to just set again the fallbacks manually? (E.g. after clean checkout/clone of the project from git.)
